### DISCUSSION TITLE
mkdocs-material/blog/2021/the-past-present-and-future/

### DISCUSSION THREAD

**Discussion by giscus** [squidfunk/mkdocs-material#3488] [_]
# mkdocs-material/blog/2021/the-past-present-and-future/

2021 was a fantastic year for this project as we shipped many new awesome features and made this project sustainable

https://squidfunk.github.io/mkdocs-material/blog/2021/the-past-present-and-future/

**Comment by JeyDi** [comment]
Your work it's amazing Martin! Please continue like that! Mkdocs it's one of the greatest tool and library in Python! :)

**Reply by squidfunk** [reply]
Thanks! I'm glad Material for MkDocs is useful to you 😊

**Reply by alanlivio** [reply]
Amazing work indeed.
